utils/build-presets.ini

```diff
@@ -2103,26 +2103,33 @@ swiftpm
 swiftsyntax
 skstresstester
 playgroundsupport
+libcxx
```

Comment:
I wonder if adding these flags increases the toolchain size. Also `install-libcxx` and additions to `swift-install-components` and `llvm-install-components` below.